js/languages/en-US.json

```diff
@@ -806,6 +806,7 @@
   },
```

Comment:
Cool. Thanks for that @morebrownies. Since it looks like just me with the issue, I won't hold up the merge.

Oops... sorry Wolf, I just merged without checking in with you. If there are any problems on your end, I'm sure Josh has another daily purchasing PR coming today or tomorrow that you could comment... lol.